README.md

```diff
@@ -17,13 +17,17 @@
    ```
 
 1. Install our ROS dependencies
+   Remember to source the underlay script corresponding to your shell (.sh, .bash, and .zsh are available). 
```

Comment:
Need a blank line between this and the numbered step to actually get rendered on a new line.